### PR TITLE
Transfer functionality

### PR DESCRIPTION
**Pull request by Arielpetit**
@Bansikah  and @Ngha-Boris  please review the Transfer-functionality to merge

### CODE REVIEW

**Review by Ngha-Boris — APPROVED** [review]
@Arielpetit i think the ticket is well implemented good job

**Review by Bansikah — COMMENTED** [review]
@Arielpetit good

File: static/style.css
Comment (Bansikah) [inline static/style.css:1]: @Arielpetit always preferable to create your own file.css and work in if the codes you gonna write is much

**Review by Bansikah — COMMENTED** [review]
@Arielpetit  Good

File: static/style.css
Comment (Arielpetit) [reply static/style.css:1]: Ok sir

**Pull request merged**